data/themes/classic/style.css

```diff
@@ -148,6 +148,7 @@ lmms--gui--PianoRoll {
 	qproperty-noteModeColor: rgb( 255, 255, 255 );
 	qproperty-noteColor: rgb( 119, 199, 216 );
 	qproperty-stepNoteColor: #9b1313;
+	qproperty-curStepNoteColor: rgb(245, 3, 139);
```

Comment:
The beat note color is named `stepNoteColor` so `curStepNoteColor` doesn't feel like the best variable name. Maybe `curStepEdit`?

Reply:
`currentStepNoteColor` maybe. The name is verbose anyway so why skimping out on characters?

Reply:
The point was that `stepNoteColor` and `curStepNoteColor` isn't logically related. The first is beat note color under the Piano Roll and the other is a step note color while being created.

Reply:
I agree with Ross, we should update the name to be not shorthand.

Reply:
It has been done.